### DISCUSSION TITLE
Why periodicRefreshEnabled defaults to False?

### DISCUSSION THREAD

**Discussion by yuandongjian** [redis/lettuce#2081] [General]
My Redis Cluster went down a node, but it quickly picked the master. If periodicRefreshEnabled is not enabled, the Lettuce command will always timeout unless the service is restarted.

so, I wonder why periodicRefreshEnabled is false by default？

**Comment by mp911de** [comment]
There's no good default for periodic refreshes. Typically, adaptive refresh is the better strategy but it always depends on the actual environment.
